modules/mobile/spec/request/appointments_create_request_spec.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'mobile/v0/vaos_appointments/appointments_helper'
+require_relative '../support/iam_session_helper'
+require_relative '../support/matchers/json_schema_matcher'
+
+RSpec.describe 'vaos appointments', type: :request, skip_mvi: true do
+  include SchemaMatchers
+  mock_clinic = {
+    'service_name': 'service_name',
+    'physical_location': 'physical_location'
+  }
+
+  mock_facility = {
+    'test' => 'test'
+  }
+
+  before do
+    Flipper.enable('va_online_scheduling')
+    allow_any_instance_of(IAMUser).to receive(:icn).and_return('1012846043V576341')
+    iam_sign_in(build(:iam_user))
+    allow_any_instance_of(VAOS::UserService).to receive(:session).and_return('stubbed_token')
+    # rubocop:disable Layout/LineLength
+    allow_any_instance_of(Mobile::V0::VAOSAppointments::AppointmentsHelper).to receive(:get_clinic).and_return(mock_clinic)
+    allow_any_instance_of(Mobile::V0::VAOSAppointments::AppointmentsHelper).to receive(:get_facility).and_return(mock_facility)
+    # rubocop:enable Layout/LineLength
+  end
+
+  before(:all) do
+    @original_cassette_dir = VCR.configure(&:cassette_library_dir)
+    VCR.configure { |c| c.cassette_library_dir = 'modules/mobile/spec/support/vcr_cassettes' }
+  end
+
+  after(:all) do
+    VCR.configure { |c| c.cassette_library_dir = @original_cassette_dir }
+
+    Flipper.disable('va_online_scheduling')
```

Comment:
made a couple changes from the original branch. wrapped the VCR config in a before(:all) though I'm not sure that would be the cause of the issue. I also disabled the va_online_scheduling. not sure if that flag is used elsewhere. Hoping the other changes made along with the original revert of this branch will solve the issue.